docs/standards/hcs-3.md

```diff
@@ -0,0 +1,70 @@
+# HCS-3 Standard: Recursion and Cross-Chain References with Hedera Consensus Service **
+
+### Status: Published
+
+### Table of Contents
+
+- [Status: Published](#status-published)
+- [Table of Contents](#table-of-contents)
+- [Authors](#authors)
+- [Abstract](#abstract)
+- [Motivation](#motivation)
+- [Specification](#specification)
+  - [Protocol Overview](#protocol-overview)
+  - [Recursion in HCS](#recursion-in-hcs)
+  - [Cross-Chain Recursion](#cross-chain-recursion)
+  - [Implementation Guidelines](#implementation-guidelines)
+  - [Security Considerations](#security-considerations)
+  - [Future Work](#future-work)
+  - [Conclusion](#conclusion)
+
+## Primary Author
+- Patches [https://twitter.com/TMCC_Patches]()
+
+## Secondary Authors
+- Kantorcodes [https://twitter.com/kantorcodes]()
+
+## Abstract
+This document introduces HCS-3, a protocol extension for the Hedera Consensus Service, enabling recursive referencing of inscriptions and cross-chain data access. It expands Hedera's capabilities to interact with other blockchain platforms, facilitating a more interconnected and versatile DLT ecosystem.
+
+## Motivation
+The aim of HCS-3 is to leverage Hedera's high-throughput, low-latency consensus mechanism for broader blockchain interoperability. This includes enabling complex data structures and references to cross-chain assets and data on traditional blockchains like Bitcoin. The protocol seeks to foster a new level of integration for decentralized applications.
+
+## Specification
+### Protocol Overview
+HCS-3 extends the existing `hcs://{protocol number}/{topic_id}` URL structure for referencing within and across blockchain networks, supporting complex data structures and cross-chain interactions.
```

Comment:
I sort of wonder if we should have a dedicated standard for HRL, and if it should be numbered before recursion is introduced. Just a trailing thought.